openwrt-openwisp-monitoring/files/monitoring.init

```diff
@@ -0,0 +1,37 @@
+#!/bin/sh /etc/rc.common
+
+START=99
+STOP=15
+USE_PROCD=1
+PROG="/usr/sbin/openwisp_monitoring"
+PROG_NAME="OpenWISP monitoring daemon"
+
+start_service() {
+		config_load openwisp
+		base_url=$(config_get http url)
```

Comment:
I'm not quite sure we should be doing it this way.  What if the value is undefined?  Instead of writing the exception handling code yourself, if you define a `config_openwisp_monitoring` function as I mention elsewhere, `procd` should automatically pass the `openwisp_monitoring` configuration (either file or UCI, I'm not sure which) as the `$1` argument and, you are then able to do something like this: `config_get period "$1" period "120"`  That's: _"assign the value in the config file to `period`, and if it is null use `120` as a default value"_